### ISSUE TITLE
Getting Audio Filled With Silence on Interference After Training Awhile, Previous Epoch Did Not Do This

### ISSUE THREAD

**Issue by SeymourNickelson** [p0p4k/vits2_pytorch#105]
So I trained up to 1609000 steps.

 I have a single word phrase in the training dataset (I have a lot of these). At this point when I run inference using this exact phrase it synthesizes fine. 

Then I stopped training. I added some more samples to the training dataset (but none of the newly added samples have this word in it) and I resumed training.

I trained up to 1962000 steps. Mel loss continues to decrease, and it looks like training is going in the right direction but I noticed now when I synthesize the exact same phrase I synthesized back at 1609000 steps I get audio filled completely with silence. 

Anyone know what could be causing this to happen? I imagine there could be more problematic phrases but I just happened to notice this one.  Before resuming training I increased max_text_len from 190 to 202. Can't think of any other changes, aside from adding more audio samples to the training set.

I can workaround the issue by adding a single whitespace character before and after the text and then synthesize...then I get audio instead of silence for the problematic phrase. Anyone know what's wrong and how I can resolve this? Thanks a lot to anyone who can help.

**Comment by SeymourNickelson** [comment]
So I think the problem was that I had a handful of samples in the dataset that had the wrong bit depth.

**Comment by lumpidu** [comment]
Hey @SeymourNickelson: could you share your XP with this model in comparison to Piper VITS ? I remember we had a small discussion about your training setup one year ago.

**Comment by SeymourNickelson** [comment]
In terms of a quality comparison it's hard for me to say because my experience with this model is still somewhat limited. The quality sounds pretty good though (again limited testing, and despite my few dirty samples). I'm not sure if I could say the quality difference between this and Piper TTS is _so_ noticeably different though (but TBH I didn't do an extensive comparison between the two).

One obvious improvement here is that Vits2 doesn't use a pad token. I recall when using piper I had to use an argument like --max-phoneme-ids and drop few longer phrases from the dataset to get training to work with my 3090. But with no pad token this wasn't an issue with Vits2. 

Advantage of piper tts is that repo has a lot of features built in (like --resume_from_single_speaker_checkpoint, quality options, etc.) which you have to reimplement here yourself.
